obspy/io/quakeml/tests/test_quakeml.py

```diff
@@ -1009,6 +1009,24 @@ def test_read_same_file_twice_to_same_variable(self):
         # No warning should have been raised.
         self.assertEqual(len(w), 0)
 
+    def test_focal_mechanism_write_read(self):
+        """
+        Test for a bug in reading a FocalMechanism without MomentTensor from
+        QuakeML file. Makes sure that FocalMechanism.moment_tensor stays None
+        if no MomentTensor is in the file.
+        """
+        memfile = io.BytesIO()
+        # create virtually empty FocalMechanism
+        fm = FocalMechanism()
+        event = Event(focal_mechanisms=[fm])
+        cat = Catalog(events=[event])
+        cat.write(memfile, format="QUAKEML")
+        # now read again, and make sure there's no stub MomentTensor, but
+        # rather `None`
+        memfile.seek(0)
+        cat = read_events(memfile, format="QUAKEML")
+        self.assertEqual(cat[0].focal_mechanisms[0].moment_tensor, None)
```

Comment:
Maybe also write it out again with `validate=True` to test the behavior because of which you discovered the bug?